quickwit-config/src/index_config.rs

```diff
@@ -188,8 +188,9 @@ impl Default for IndexingSettings {
     }
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Default)]
```

Comment:
I think `Default` on `SearchSettings` is dangerous. That means you can index some docs with the minimal config and have all the subsequent searches on that index come back empty because the default search fields are not set.

Reply:
I don't really agree. I would be happy to create an index without this parameter, no default search field is fine, it's not q a required setting.

Reply:
I don't understand how our search works then. A `SearchRequest` has an optional `search_fields` field; what happens if this field is not specified by the user?

Reply:
the typical search query is like this `my_field:value`